src/entities/PricebookEntry.js

```diff
@@ -0,0 +1,32 @@
+const Entity = require('./Entity');
+class PricebookEntry extends Entity{
+    price;
+    constructor(pricebookId, productId) {
+        super();
+        if(typeof pricebookId == undefined || typeof productId == undefined) {
+            throw new Error(`Both Pricebook and Product Id's are required to instanciate a PricebookEntry`);
```

Comment:
I don't think this is not a good way for error handling. Entities should not handle errors.

Reply:
Understood. I'll remove this. TY!